diff --git a/miniep2/src/Makefile b/miniep2/src/Makefile
index d0aedde..4e5188d 100644
--- a/miniep2/src/Makefile
+++ b/miniep2/src/Makefile
@@ -1,5 +1,5 @@
 CC     = gcc
-CFLAGS = -Wall -pedantic -g -O2 -flto
+CFLAGS = -Wall -pedantic -g -O2
 RM     = rm -rf
 print  = echo
 
diff --git a/miniep2/src/main.c b/miniep2/src/main.c
index e48dc51..bb56334 100644
--- a/miniep2/src/main.c
+++ b/miniep2/src/main.c
@@ -103,6 +103,8 @@ int main(int argc, const char* argv[])
     matrix_fill_rand(n, A);
     matrix_fill_rand(n, B);
 
+    memset(C, 0x00, n*n*sizeof(*C));
+
     gettimeofday(&t1, NULL);
     if (!matrix_which_dgemm(args->algorithm, n, C, A, B))
     { 
